packages/fxa-settings/src/pages/ResetPasswordRedesign/CompleteResetPassword/en.ftl

```diff
@@ -11,3 +11,5 @@ complete-reset-password-success-alert = Password set
 complete-reset-password-error-alert = Sorry, there was a problem setting your password
 complete-reset-password-recovery-key-error-v2 = Sorry, there was a problem checking if you have an account recovery key.
 complete-reset-password-recovery-key-link = Reset your password with your account recovery key.
+
+account-restored-success-message = You have successfully restored your account using your account recovery key. Create a new password to secure your data, and store it in a safe location.
```

Comment:
Since I didn't do this myself, did you go through the flows with the console pulled up to double check there aren't any missing FTL IDs? Good catch on moving this one over.

Reply:
We should be good, I moved the shared FTL files over. `AccountRecoveryResetPassword` and `CompleteResetPassword` are now a shared component so I combined the FTL files.

We have a separate ticket filed to remove the feature flags (there are two, one for auth and one for content-server/settings) (but I'll add a note there to also remove the SRE flags)